frontend/src/lib/components/NavPanelAdvertisement/NavPanelAdvertisement.tsx

```diff
@@ -0,0 +1,136 @@
+    const { hideAdvertisement } = useActions(logic)
+    const { hidden } = useValues(logic)
+
+    if (hidden || (!product && !payload.header)) {
+        return null
+    }
+
+    return (
+        <div className="w-full my-1">
+            <Tooltip
+                title="Based on your usage we think you might like this other product
+                            we have, so we're gently bringing it to your attention :) Feel free to 
+                            dismiss it to make it go away."
+            >
+                <p className="text-xxs text-muted mb-1 p-0 text-right">ads via PostHog</p>
+            </Tooltip>
+            <Link
+                to={payload.app_link}
+                className="text-primary"
+                onClick={() => {
+                    posthog.capture('nav panel advertisement clicked', {
+                        product_key: payload.product_key,
+                        payload,
+                    })
+                }}
+            >
+                <div className="border rounded bg-primary text-xs *:flex *:gap-2 *:px-2 *:py-1">
+                    <div className=" justify-between mt-1">
+                        <div className="flex items-center gap-2">
+                            {product && getProductIcon(product.iconColor, product.icon, 'text-lg')}
+                            <strong>{payload.header || product?.name}</strong>
+                        </div>
+                        <LemonButton
+                            icon={<IconX className="text-muted" />}
+                            tooltip="Dismiss"
+                            tooltipPlacement="right"
+                            size="xxsmall"
+                            onClick={(e) => {
+                                e.preventDefault()
+                                e.stopPropagation()
+                                posthog.capture('nav panel advertisement dismissed', {
+                                    product_key: payload.product_key,
+                                    payload,
+                                })
+                                hideAdvertisement()
+                            }}
+                            noPadding
+                        />
+                    </div>
+
+                    <div className="flex flex-col gap-1">
+                        <p className="mb-0">{payload.text}</p>
+                        <p className="mb-0">
+                            <Link
+                                onClick={(e) => {
+                                    e.preventDefault()
+                                    posthog.capture('nav panel advertisement learn more clicked', {
+                                        product_key: payload.product_key,
+                                        payload,
+                                    })
+                                    window.open(payload.product_info_link, '_blank')
```

Comment:
Why not simply use the `to` prop on the button? Then you don't need to call `e.preventDefault` or anything, and we'll also properly respect the behavior of opening this on a new PostHog tab rather than a separate tab.

Reply:
Ah, maybe because we don't support `_blank` with that? Should probably add that behavior instead

Reply:
no it's because I want the whole card to be clickable, so the whole thing is wrapped in an Link, but the X button needs to NOT do anything that would normally happen (specifically calling the "ed clicked" event) and this was the best way to do it